src/Microsoft.DotNet.Build.Tasks/PackageFiles/xunit.runner.json

```diff
@@ -1,4 +1,5 @@
 {
     "diagnosticMessages": true,
-    "longRunningTestSeconds": 120
+    "longRunningTestSeconds": 120,
+    "shadowCopy": false
```

Comment:
I thought this setting was ignored for .NET Core?

Reply:
Right, but we use it for netfx as we also use that config for the runs there.